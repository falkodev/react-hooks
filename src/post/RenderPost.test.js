import React from 'react'
import { render } from 'react-testing-library'
import Post from './RenderPost'

it('renders a post and displays content as markdown', () => {
  const post = {
    number: 1,
    content: '**test**',
    tags: ['test1', 'test2'],
    author: 'Test Test',
  }
  const { container, getByText } = render(<Post post={post} />)
  expect(container.firstChild).toMatchSnapshot()
  expect(getByText('test')).toBeInTheDocument()
})
