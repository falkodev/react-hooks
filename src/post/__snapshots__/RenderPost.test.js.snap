// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a post and displays content as markdown 1`] = `
<div
  class="post"
>
  <div
    class="post__content"
  >
    <div>
      <span>
        <p>
          <strong>
            test
          </strong>
        </p>
        

      </span>
    </div>
  </div>
  <div
    class="post__tags"
  >
    test1
    test2
  </div>
  <div
    class="post__author"
  >
    Test Test
  </div>
</div>
`;
